DANS-KNAW/easy-download
# EASY-2464: Voeg een 'statistics-logger' toe aan easy-download

Fixes EASY-2464

#### When applied it will
* write download event into easy-statistics log-file


#### Where should the reviewer @DANS-KNAW/easy start?

#### How should this be manually tested?
* download a file from deasy, e.g. curl http://deasy.dans.knaw.nl:20160/ark:/73189/[bag-id]/data/a/deeper/path/With%20some%20file.txt
* in deasy check the contents of `/var/opt/dans.knaw.nl/log/easy-download/easy-statistics.log`
#### Related pull requests on github
repo                       | PR                | note
-------------------------- | ----------------- | ----
easy-dtap                      | [PR#430](https://github.com/DANS-KNAW/easy-dtap/pull/430)     | ..
